frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -120,16 +126,26 @@ function ReaderUserHyph:updateDictionary(word, hyphenation)
         return
     end
 
+    if word then
+        word = Utf8Proc.normalize_NFC(word)
+    end
+
     local word_lower = Utf8Proc.lowercase(word)
     local line
 
     local dict = io.open(dict_file, "r")
     if dict then
         line = dict:read()
+        if line then
+            line = Utf8Proc.normalize_NFC(line)
+        end
```

Comment:
Would `line = line and Utf8Proc.normalize_NFC(line)` just work (and keep all these single line) ?